src/main/java/uk/gov/hmcts/cmc/claimstore/config/FeignClientConfiguration.java

```diff
@@ -32,4 +36,27 @@ public Client getFeignHttpClient() {
             .setDefaultRequestConfig(config)
             .build());
     }
+
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
This isn't about FeignClientConfiguration, maybe rename the class to be about http?

Reply:
That's been done.